Neos.Utility.Arrays/Classes/Arrays.php

```diff
@@ -25,11 +25,11 @@
      * @param string $string The string to explode
      * @return array Exploded values, all converted to integers
      */
-    public static function integerExplode($delimiter, $string)
+    public static function integerExplode(string $delimiter, string $string): array
     {
         $chunksArr = explode($delimiter, $string);
-        while (list($key, $value) = each($chunksArr)) {
-            $chunks[$key] = intval($value);
+        foreach ($chunksArr as $key => $value) {
```

Comment:
A foreach is a lot faster than the previous implementation. Was there a specific reason for choosing that?